lib/navigation/app_controller.dart

```diff
@@ -23,43 +23,53 @@ class AppController extends StatefulWidget {
 class _AppControllerState extends State<AppController> {
 
   final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+  final List<Widget> _screens = [UserProfile(), Pairings(), Chat()];
   int pageIndex = 1; 
-  double navBarIconSize = 30; 
-  final List<Widget> _widgets = [UserProfile(), Pairings(), Chat()];
+  double navBarIconSize = 25.0; 
+  Color navItemColour = Color(0xFF44CBB9);
```

Comment:
What do we want to do about the theme colours? Place it in a constants file?

Reply:
Sounds good to me!

Reply:
Theme colours should be in a constants file but then shouldn't we be getting those colours from the context?

Reply:
Do you mean define the theme at the top level of our app?